Greentube.Monitoring.sln

```diff
@@ -47,80 +47,80 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{2BB29E61-02DD-4745-A3A2-145711D5FFB8}.Coverage|Any CPU.ActiveCfg = Coverage|Any CPU
-		{2BB29E61-02DD-4745-A3A2-145711D5FFB8}.Coverage|Any CPU.Build.0 = Coverage|Any CPU
+		{2BB29E61-02DD-4745-A3A2-145711D5FFB8}.Coverage|Any CPU.ActiveCfg = Release|Any CPU

```

Comment:
Why that was changed? 
Don't really know what it affects, have to check

Reply:
Also don't know, but checked on another dev machine and it's the same, full rebuild update .sln

So it's better to keep those changes

Reply:
coverage config is added to csproj, now it don't disappear.